### PR TITLE
Fix operating computer automatic linking (+ examine tweaks)

### PR DESCRIPTION
**Pull request by Absolucy**
## About The Pull Request

This fixes operating computer automatic linking not working, because I made a typo with variable names (although if you build the stasis bed / op table _after_ the computer, it'd auto-link just fine)

## Why It's Good For The Game

BUGFIX WAAAAA

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
fix: Fix operating computer automatic linking only working if you built the computer first, and the table/bed last.
add: Operating tables now display when examined if they're linked to a computer or not (and if they are, which computer).
tweak: Stasis beds now display WHICH operating computer they're linked to when examined
/:cl:

### CODE REVIEW

File: code/game/machinery/computer/Operating.dm [hunk 1]
```diff
@@ -52,13 +52,13 @@
 	for(var/direction in GLOB.alldirs)
 		var/obj/structure/table/optable/found_table = locate(/obj/structure/table/optable) in get_step(src, direction)
 		if(found_table && (!found_table.computer || found_table.computer == src))
-			return table
```
Comment (Absolucy) [inline code/game/machinery/computer/Operating.dm:55]: such an easy-to-miss yet devastating (well, as in, it broke the whole functionality of this proc) typo, lol

**Comment by itsmeow** [comment]
HOW IS THIS NOT FIXED.

**Comment by itsmeow** [comment]
GRAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

**Comment by Tsar-Salat** [comment]
ISNF THIS YOUR FOURTH PR FOR THIS

I THOUGHT I WAS BAD

**Comment by Absolucy** [comment]
I MADE A TYPO AND ONLY TESTED THE SPECIFIC CASE I WAS TRYING TO FIX WHILE NOT NOTICING I BROKE OTHER CASES 😭

**Comment by itsmeow** [comment]
WRITE ME A UNIT TEST RN

**Comment by Absolucy** [comment]
i don't know how ss13 unit tests work

**Comment by itsmeow** [comment]
it's DM, and it either passes or fails. You are given a small semi-physical space run procs in.

**Comment by Absolucy** [comment]
@itsmeow unit test added!

**Comment by itsmeow** [comment]
Holy shit you actually did it

**Review by itsmeow — APPROVED** [review]

**Comment by Tsar-Salat** [comment]
Incredible

**Pull request merged**